websites/C/CodinGame/presence.ts

```diff
@@ -0,0 +1,199 @@
+			presenceData.details = "Training: Code Golf";
+			try {
+				if (
+					document.querySelector("div[class='chart-inner']").textContent !== "0"
+				) {
+					presenceData.state = `${
+						document.querySelector("h1[class='title']").textContent
+					} (${document.querySelector("span[class='rank-value']").textContent}${
+						document.querySelector("span[class='rank-suffix']").textContent
+					}/${document.querySelector("span[class='rank-total']").textContent})`;
+				} else presenceData.state = "Not yet solved";
+			} catch {
+				presenceData.state = `${
+					document.querySelector("h1[class='title']").textContent
+				} (${document.querySelector("span[class='rank-value']").textContent}${
+					document.querySelector("span[class='rank-suffix']").textContent
+				}/${document.querySelector("span[class='rank-total']").textContent})`;
+			}
+		} else if (path.includes("/multiplayer/optimization")) {
+			presenceData.details = "Training: Optimization";
+			try {
+				if (
+					document.querySelector("div[class='chart-inner']").textContent !== "0"
+				) {
+					presenceData.state = `${
+						document.querySelector("h1[class='title']").textContent
+					} (${document.querySelector("span[class='rank-value']").textContent}${
+						document.querySelector("span[class='rank-suffix']").textContent
+					}/${document.querySelector("span[class='rank-total']").textContent})`;
+				} else presenceData.state = "Not yet solved";
+			} catch {
+				presenceData.state = `${
+					document.querySelector("h1[class='title']").textContent
+				} (${document.querySelector("span[class='rank-value']").textContent}${
+					document.querySelector("span[class='rank-suffix']").textContent
+				}/${document.querySelector("span[class='rank-total']").textContent})`;
+			}
+		} else presenceData.details = "Looking at Compete";
+	} else if (path.includes("/cooperate"))
+		presenceData.details = "Looking at Cooperate";
+	else if (path.includes("/learn")) presenceData.details = "Looking at Learn";
+	else if (path.includes("/events")) presenceData.details = "Looking at Events";
+	else if (path.includes("/contribute/community"))
+		presenceData.details = "Looking at Community";
+	else if (path.includes("/contribute/my-contributions"))
+		presenceData.details = "Looking at My Contributions";
+	else if (path.includes("/live-streams"))
+		presenceData.details = "Looking at Live Streams";
+	else if (path.includes("/blog")) presenceData.details = "Looking at Blog";
+	else if (path.includes("/forum")) presenceData.details = "Looking at Forum";
+	else if (path.includes("/profile"))
+		presenceData.details = "Looking at Profile";
+	else if (path.includes("/settings"))
+		presenceData.details = "Looking at Settings";
+	else if (path.includes("/about")) presenceData.details = "Looking at Abouts";
+	else if (path.includes("/careers"))
+		presenceData.details = "Looking at Careers";
+	else if (path.includes("/work")) presenceData.details = "Looking at Work";
```

Comment:
Include your articles
```suggestion
	else if (path.includes("/learn")) presenceData.details = "Looking at the learn page";
	else if (path.includes("/events")) presenceData.details = "Looking at the events page";
	else if (path.includes("/contribute/community"))
		presenceData.details = "Looking at community page";
	else if (path.includes("/contribute/my-contributions"))
		presenceData.details = "Looking at their contributions";
	else if (path.includes("/live-streams"))
		presenceData.details = "Looking at live streams";
	else if (path.includes("/blog")) presenceData.details = "Looking at the blog";
	else if (path.includes("/forum")) presenceData.details = "Looking at the forum";
	else if (path.includes("/profile"))
		presenceData.details = "Looking at a profile";
	else if (path.includes("/settings"))
		presenceData.details = "Looking at their settings";
	else if (path.includes("/about")) presenceData.details = "Looking at the about page";
	else if (path.includes("/careers"))
		presenceData.details = "Looking at the careers page";
	else if (path.includes("/work")) presenceData.details = "Looking at the work page";
```
Only accept the suggestion if the suggestion is fitting for the situations